src/current/cockroachcloud/connect-to-your-cluster.md

```diff
@@ -36,23 +36,57 @@ Removing or adding an authorized network on your CockroachDB {{ site.data.produc
 
 1. Click **Apply**.
 
-### Establish GCP VPC Peering or AWS PrivateLink
+<a id="establish-gcp-vpc-peering-or-aws-privatelink"></a>
+### Establish private connectivity
 
-GCP VPC Peering and AWS PrivateLink allow customers to establish SQL access to their clusters entirely through cloud provider private infrastructure, without exposure to the public internet, affording enhanced security and performance.
+Private connectivity allows you to establish SQL access to a CockroachDB {{ site.data.products.dedicated }} cluster entirely through cloud provider private infrastructure, without exposing the cluster to the public internet, affording enhanced security and performance.
 
-VPC peering is available only for GCP clusters, and AWS PrivateLink is available for AWS clusters.
+- Clusters deployed on GCP can connect privately using [GCP Private Service Connect (PSC)](#gcp-private-service-connect) or [GCP VPC peering](#gcp-vpc-peering). PSC allows you to selectively connect your cluster to a VPC within your Google Cloud project, while VPC Peering allows you to connect the Cockroach Cloud's VPC for your cluster to a VPC within your Google Cloud project.
+- Clusters deployed on AWS can connect privately using AWS PrivateLink, which allows you to connect Cockroach Cloud's VPC to a VPC within your AWS account.
 
-To configure VPC Peering or PrivateLink, you create the private connection in your cloud provider, then configure your cluster to allow connections from your VPC or private endpoint. For more information, refer to [Network Authorization for CockroachDB {{ site.data.products.dedicated }} clusters: GCP VPC Peering]({% link cockroachcloud/network-authorization.md %}#vpc-peering) and [Network Authorization for CockroachDB {{ site.data.products.dedicated }} clusters: AWS PrivateLink]({% link cockroachcloud/network-authorization.md %}#aws-privatelink).
+For more information, refer to [Network authorization]({% link cockroachcloud/network-authorization.md %}).
 
-AWS PrivateLink can be configured only after the cluster is created. For detailed instructions, refer to [Managing AWS PrivateLink for a cluster]({% link cockroachcloud/aws-privatelink.md %}). To configure VPC Peering, continue to the [VPC Peering](#vpc-peering) section below.
+{{site.data.alerts.callout_success}}
+GCP Private Service Connect and AWS PrivateLink can be configured only after a cluster is created.
+{{site.data.alerts.end}}
 
 Azure Private Link is not yet available for [CockroachDB {{ site.data.products.dedicated }} on Azure]({% link cockroachcloud/cockroachdb-dedicated-on-azure.md %}).
 
 {{site.data.alerts.callout_info}}
 {% include cockroachcloud/cdc/kafka-vpc-limitation.md %}
 {{site.data.alerts.end}}
 
-#### VPC Peering
+#### GCP Private Service Connect
+
+{{site.data.alerts.callout_info}}
+{% include_cached feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+1. Navigate to your cluster's **Networking > Private endpoint** tab.
```

Comment:
not for this PR thoughts:

as a user I'd love these "your cluster's FOO tab" mentions to all be links to CC console things

maybe not possible unless we have some kind of SSO across docs site / console

one day maybe we'll ship these docs IN console and solve this